### DISCUSSION TITLE
Hide bidirectional relationship field from Flexible Relationships 'Add New'

### DISCUSSION THREAD

**Discussion by CPapadim** [pods-framework/pods#6755] [Ideas]
I'm using a bidirectonal relationship with the Add New flexible relationships functionality.  So in an example with books relating to authors, and authors relating to books, users can edit a book, adding new authors.  They can also edit authors, adding new books.  

The issue is when a user is editing an author, they might 'Add New' to create a new book for this author.  This book has a bidirectional relationship back to the author.  So now as a user is adding the new book, there's confusion - should the author also be entered, since books have author as a field too?

In other words it creates an Edit Author -> Edit Book -> Edit Author situation.  It would be nice to be able to hide the edit author from the book content type when the book is being edited in the modal window launched during editing the Author.

I hope that makes sense, it's quite circular and hard to put into words :)

**Comment by sc0ttkclark** [comment]
I like this idea, our team has talked about it a little bit already before.

There's a few complexities involved as well to tackle:

* What if the modal is being shown for something that doesn't exist yet, since only posts get special treatment in WordPress when adding a new one? (Terms, Users, Advanced Content Types, etc -- only posts get created as the page loads when you go to add a new one)
* What if the bi-di field is required in the form in the modal?
* What if they add that item through the modal but decide not to save the changes on the screen they are on?

I like the idea though, I think we should probably hide the field if we detect the bi-di in general, but we can see what @pglewis thinks too. I don't want to distract him from the other fixes he's working on for our final Pods 2.7 release right now, but perhaps he can look at this when we go through 2.7.1 and 2.8 planning.
